engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -162,11 +159,14 @@ IR.Module translateModule(Tree module) {
         var args = translateArgumentsDefinition(fn.getArgs());
         var body = translateExpression(fn.getBody());
 
+        if (body == null) {
+            throw new NullPointerException();
```

Comment:
Maybe throw a more informative exception? This seems like an internal error, no?

Reply:
The information is the stack trace!

Previously the execution continued and later, when certain compiler phase tried to traverse the `IR`, another `NullPointerException` was thrown. That one was however non-informative. Throwing the exception early on helps to identify the source of the problem.